kerascv/layers/ssd_box_coder.py

```diff
@@ -0,0 +1,94 @@
+import tensorflow as tf
+
+
+class SSDBoxCoder(tf.keras.layers.Layer):
+    """Defines a SSDBoxEncoder that converts encodes the ground_truth_boxes using anchors.
+
+    Mathematically, the encoding is:
+        $ \hat{cx_gt} = (cx_gt - cx_a) / width_a
+        $ \hat{cy_gt} = (cy_gt - cy_a) / height_a
+        $ \hat{width_gt} = log(width_gt / width_a)
+        $ \hat{height_gt} = log(height_gt / height_a)
+
+    where cx, cy, width, height represents center of width, center of height, width, height respectively,
```

Comment:
Do you have a coding style setting up? It looks this line is too long.

Reply:
All that's why :-) Done